libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFNetwork.m

```diff
@@ -97,4 +141,53 @@ + (void)setSessionManager:(id<SFNetworkSessionManaging>)manager {
     kSFNetworkManager = manager;
 }
 
++ (NSDictionary *)sharedSessions {
+    return [[self mutableSharedSessions] dictionary];
+}
+
++ (void)removeSharedEphemeralSession {
+    [[self class] removeSharedSessionForIdentifier:kSFNetworkEphemeralSessionIdentifier];
+}
+
++ (void)removeSharedBackgroundSession {
+    [[self class] removeSharedSessionForIdentifier:kSFNetworkBackgroundSessionIdentifier];
+}
+
++ (void)removeSharedSessionForIdentifier:(nullable NSString *)identifier {
+    if (identifier) {
+        [[self mutableSharedSessions] removeObject:identifier];
+    }
+}
+
++ (void)removeAllSharedSessions {
+    [[self mutableSharedSessions] removeAllObjects];
```

Comment:
Remove usage of `self` from this class method. This should access the `static` dictionary directly.